Makefile.am

```diff
@@ -71,6 +71,8 @@ enum_sources = \
 	libeos-updater-util/enums.c \
 	$(NULL)
 
+# We can't use the --output switch as documented in https://git.gnome.org/browse/glib/commit/?id=cd97f93bf7c44cb5736f6cbcb4db773ca592e462
```

Comment:
Nitpick: Please start this comment with `FIXME:` then we can easily grep for it and find it later. (i.e. `git grep -i fixme` becomes a kind of to-do list for the module.)